notebooks/ml/evaluation.ipynb

```diff
@@ -10,7 +10,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
I was quite surprised to find out that we are currently not running the `nb-clean` pre-commit hook in this repository to clean up the `jupyter` notebooks. https://github.com/srstevenson/nb-clean
First of all, this makes the notebooks way leaner. But it also strips away some of the personal information from running the notebook. There seem to even be options for preserving some of the outputs if that is desired.
However, I believe people who are really interested in the results should replicate them on their own machine anyway.
Overall, I'd be in favor of adopting the pre-commit check as shown here: https://github.com/cda-tum/qcec/blob/e7c77398168647736cf7e309fe0668c20773da3a/.pre-commit-config.yaml#L145-L149

Reply:
Very good suggestion, thanks. I added it accordingly.